package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "bulib-webcomponents",
+  "version": "0.9.0",
+  "description": "collection of web components and helpers used at Boston University Libraries",
+  "dependencies": {
+    "@webcomponents/webcomponentsjs": "^2.2.7",
+    "lit-element": "^2.0.1"
+  },
+  "devDependencies": {
+    "@open-wc/building-webpack": "^0.1.9",
+    "@open-wc/polyfills-loader": "^0.2.2",
+    "http-server": "^0.11.1",
+    "owc-dev-server": "^0.1.6",
+    "polymer-cli": "^1.9.6",
+    "webpack": "^4.29.5",
+    "webpack-dev-server": "^3.2.0",
+    "webpack-cli": "^3.2.3"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bulib/bulib-wc.git"
+  },
+  "keywords": [
+    "web-components",
+    "lit-element",
+    "lit-html",
+    "polymer",
+    "code4lib"
+  ],
+  "author": "@bulib",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/bulib/bulib-wc/issues"
+  },
+  "homepage": "https://github.com/bulib/bulib-wc#readme",
+  "scripts": {
+    "build": "webpack ./index.js --output ./build/bundle.js",
+    "build:es5": "webpack ./index.js --output ./build/bundle_es5.js --mode production --es5",
+    "build:es6": "webpack ./index.js --output ./build/bundle_es6.js --mode production --es6",
```

Comment:
- it's almost certainly the case that the `--es5` and `--es6` flags are not doing anything, but this demonstrates the desire to have multiple output files 
- the running of each of these commands inexplicably creates a dozen additional files (`[0-12].bundle.js`). this is likely due to ['code splitting'](https://webpack.js.org/guides/code-splitting/), but is not desired. not sure how this works or how to prevent it.